src/content/docs/full-stack-observability/observe-everything/get-started/get-started-auto-telemetry-pixie.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Get started with Auto-telemetry with Pixie
+tags:
+  - Pixie Auto-telemetry
+  - Service monitoring
+  - Kubernetes
+  - eBPF
+metaDescription: 
+watermark: BETA
+redirects:
+  - /docs/full-stack-observability/get-started-auto-telemetry-pixie
+---
+
+<Callout variant="important">
+Auto-telemetry with Pixie is a supported public beta. Therefore, the features and functionality are subject to change. By joining the beta for Auto-telemetry with Pixie, you agree to the terms in the [New Relic Pre-Release Policy](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy/). 
+</Callout>
+ 
+When we say auto-telemetry, we’re not talking about cars &mdash; we’re talking about automatically collecting telemetry data for your Kubernetes clusters. With our [Pixie](https://pixielabs.ai/) integration, you no longer have to manually instrument your clusters or change code. With one install command, you get observability into your Kubernetes cluseters and workloads. No language agents required. 
+
+Pixie collects telemetry data using [eBPF](https://ebpf.io/), a virtual machine-like construct in the Linux kernel. eBPF makes it easy to collect similar data about your clusters and workloads as with traditional New Relic APM agents, but now without needing to update your code.
+
+## Why it matters 
+
+Auto-telemetry with Pixie offers the quickest option for getting observability into your Kubernetes services. Did we mention no language agents are required?  
+
+Our Pixie integration gives you the best of both worlds: Pixie’s fast and simple Kubernetes observability coupled with New Relic One’s incident correlation, intelligent alerting, and long-term retention. You’ll get visibility into HTTP services using golden signals, HTTP transactions, database transactions, distributed tracing, and JVM metrics. You can operate, debug, and scale your Kubernetes clusters based on the information you learn about how your clusters and services are running. Using the New Relic [Explorer](/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts/), you can see key metrics and events at every level, starting with the cluster, and diving down into namespaces, deployments, and pods. You can quickly spot anomalous behavior, and where it’s happening. If you need further insight, you can easily link over to Pixie's native UI for advanced debugging with PXL scripts. 
```

Comment:
I sort of want to swap this paragraph so that it's first, before the auto-telemetry stuff.

Reply:
There's so much here! It might be worth thinking about whether there's a way to group these things into different categories.

Is it possible to group these like so (not literally this):
You'll get incident correlation, which means x, y, and z.
You'll get intelligent alerting, which means x, y, and z.
etc

It all seems like good stuff, but my feeling about this paragraph is that it's just a stew, with lots of lovely carrots and potatoes swirling around. I guess I'd like to see things in more clearly laid out in front of me.